assemblyline_v4_service/common/base.py

```diff
@@ -23,6 +23,16 @@
 UPDATES_DIR = os.environ.get('UPDATES_DIR', '/updates')
 PRIVILEGED = os.environ.get('PRIVILEGED', 'false') == 'true'
 
+RECOVERABLE_RE_MSG = ["cannot schedule new futures after interpreter shutdown", "can't register atexit after shutdown"]
```

Comment:
can't register atexit after shutdown -> can't register **at exit** after shutdown

Reply:
weirdly enough the error is actually: can't register atexit after shutdown

Least from the logs that are recorded.